addons/calendar/models/calendar_event.py

```diff
@@ -925,13 +925,13 @@ def action_unlink_event(self, attendee_id=None, recurrence=False):
                 'views': [(False, 'form')],
             }
 
-    @api.model
-    def _get_mail_message_access(self, res_ids, operation, model_name=None):
-        if operation == 'read' and (not model_name or model_name == 'event.event'):
```

Comment:
This used to be an override only for the read operation, it seems to have been removed in the diff.